### ISSUE TITLE
Whitespaces check, inccorect hits?

### ISSUE THREAD

**Issue by martinvw** [openhab/static-code-analysis#250]
Currently the openHAB build is broken with the following report:

```
[ERROR] Code Analysis Tool has found: 
 1 error(s)! 
 3 warning(s) 
 2 info(s)
[ERROR] org.openhab.binding.onewiregpio.handler.OneWireGPIOHandler.java:[155]
Count of placeholder(2) is not equal to count of parameter(1)
[WARNING] .binding.onewiregpio/ESH-INF/thing/thing-types.xml:[3]
There were whitespace characters used for indentation. Please use tab characters instead
[WARNING] .binding.onewiregpio/ESH-INF/binding/binding.xml:[3]
There were whitespace characters used for indentation. Please use tab characters instead
[WARNING] .binding.onewiregpio/pom.xml:[3]
There were whitespace characters used for indentation. Please use tab characters instead
[INFO] Detailed report can be found at: file:////home/travis/build/openhab/openhab2-addons/addons/binding/org.openhab.binding.onewiregpio/target/code-analysis/report.html
```

But looking at the two ESH-INF files I do not see any problem (I also tried auto-formatting with eclipse), can you please validate what could be wrong? Thanks!

[binding.xml.txt](https://github.com/openhab/static-code-analysis/files/1670850/binding.xml.txt)
[thing-types.xml.txt](https://github.com/openhab/static-code-analysis/files/1670851/thing-types.xml.txt)

**Comment by martinvw** [comment]
I found the problems, see my comments at https://github.com/openhab/static-code-analysis/pull/235/files#r164292042

**Comment by MS1982** [comment]
I have the same issue in my current build process for my binding development.

**Comment by kssimova** [comment]
@MS1982 the problem with the build is fixed with https://github.com/openhab/openhab2-addons/commit/c2c557bdae414fdba2da85498e493ad8d14f09c0

@martinvw The error was caused because of mismatching count of placeholders and parameters, as you pointed out. In the release of the tool we added a detector (#237) for these cases and I fixed the problems, which occured after it with PRs in OpenHAB2-addons and ESH. This has nothing to do with the OnlyTabIdentationInXmlFilesCheck. I will take a look at the problems with the identation.  Thanks for fixing the problem found by the detector!
